staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/downloader.go

```diff
@@ -112,6 +113,23 @@ func (s *Downloader) Download(handler http.Handler, etag string) (returnSpec *sp
 	writer := newInMemoryResponseWriter()
 	handler.ServeHTTP(writer, req)
 
+	// single endpoint not found/registered in old server, try to fetch old endpoint
```

Comment:
Is there a test that this fallback works?

Reply:
Tests for fallback in downloader and discovery_client added